eva-ingest-remapped-variants/src/test/resources/input-files/vcf/vcf_out_with_header.vcf

```diff
@@ -0,0 +1,5 @@
+##fileformat=VCFv4.3
+##contig=<ID=chr2>
+#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
+chr2	98	ss5000000000	C	CG	50	PASS	PROJECT=projectId_1
+chr2	1078	ss5000000001	A	G	50	PASS	PROJECT=projectId_1
```

Comment:
Can you add an RSid in the info field I think it should be read and populated as well.

Reply:
Done